ui/src/app/integrations/list/list.component.scss

```diff
@@ -1,6 +1,7 @@
 @import 'patternfly-sass/assets/stylesheets/patternfly/color-variables';
 .integrations.list {
   color: #656565;
```

Comment:
Harcoded hex colors in component stylesheets make application theming more complex upon time. Don't we have a centralized `_palette.scss` partial or something? I'd move this one there.

Reply:
We do on the line above.  I don't even recall what this is for and it doesn't match any patternfly colors so I'll take it out.